src/ctl/cart_ctl.c

```diff
@@ -310,7 +314,9 @@ parse_args(int argc, char **argv)
 		D_GOTO(out, rc = -DER_INVAL);
 	}
 
-	if (strcmp(argv[1], "list_ctx") == 0)
+	if (strcmp(argv[1], "get_uri_cache") == 0)
```

Comment:
Not a regression but why isn't this code using getopt?

Reply:
this option is specified without the "-" or "--", getopt won't work in that case. for example: cart_ctl list_ctx